build-template/build.sh

```diff
@@ -1,7 +1,7 @@
-#!/bin/bash
+#!/bin/sh
 
 if test `uname` = Darwin; then
-    cachedir=~/Library/Caches/KBuild
+    cachedir=$HOME/Library/Caches/KBuild
```

Comment:
I thought this is the only thing holding it back from being (portable) sh. But you are right, bash is much polished standard implementation, although on some systems, bash is not available by default http://stackoverflow.com/a/5725402
#! ${Rebased}